deploy/ansible/roles/snapshotter/templates/goshimmer_snapshot.bin_dump.sh.j2

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-HTTP_STS_CODE="$(/usr/bin/curl -s -o /dev/null -w '%{http_code}\n' http://ressims.iota.cafe:9101/healthz)"
-LATEST_SNAPSHOT="http://127.0.0.1:9101/snapshot"
+HTTP_STS_CODE="$(/usr/bin/curl -s -o /dev/null -w '%{http_code}\n' http://127.0.0.1:8080/healthz)"
```

Comment:
Why it's the localhost instead of `ressims.iota.cafe`?

Reply:
I don't think we use ressims for anything DevNet related don't we?

Reply:
We want to check that the node we are going to get the snapshot from is in sync: localhost. Ressims does not run anything related to our networks anymore.